src/components/License/License.vue

```diff
@@ -85,3 +75,10 @@ export default {
   },
 }
 </script>
+
+<style scoped>
+.v-icon {
+  height: 1.1667em;
+  width: 1.1667em;
+}
+</style>
```

Comment:
Instead of rely-ing on the `VIcon` component implementation details (the `v-icon` class) can we just create a new local class to pass to the component in this template? Vue will forward the classes and merge them together.

Reply:
Great suggestion, implemented the change in edafcbd.